source/Marquee.js

```diff
@@ -232,6 +232,17 @@
 		* @method
 		* @private
 		*/
+		destroy: enyo.inherit(function (sup) {
+			return function () {
+				moon.Marquee.setMarqueeOnHoverControl(null);
```

Comment:
As in, unlike when we detect 'leave' and we know we were the control, being destroyed doesn't imply the same.